spring-cloud-gcp-pubsub-stream-binder/src/test/java/com/google/cloud/spring/stream/binder/pubsub/PubSubEmulator.java

```diff
@@ -55,9 +56,7 @@
  */
 public class PubSubEmulator implements BeforeAllCallback, AfterAllCallback, ParameterResolver {
 
-  private static final Path EMULATOR_CONFIG_DIR =
-      Paths.get(System.getProperty("user.home"))
-          .resolve(Paths.get(".config", "gcloud", "emulators", "pubsub"));
+  private static final Path EMULATOR_CONFIG_DIR = getEmulatorConfigDir();
```

Comment:
Was this passing integration tests prior to the update? If so, what's changed that now requires the OS check?

Reply:
These changes are only meant for support the ITs running in a windows environment. The point is that the gcloud config dir is in a different location on Windows. Before this change, the emulator assumes the environment is UNIX-based (mac, linux).